pkg/pwgen/pwgen_test.go

```diff
@@ -40,11 +40,14 @@ func TestPwgenCharset(t *testing.T) {
 }
 
 func TestPwgenNoCrand(t *testing.T) {
-	old := rand.Reader
-	rand.Reader = strings.NewReader("")
-
+	old := crand.Reader
+	crand.Reader = strings.NewReader("")
+	oldrand := gr
+	// if we seed math/rand with 1789, the first "random number" will be 42
+	gr = rand.New(rand.NewSource(1789))
```

Comment:
Hence the global variable. I understand better why you need one below, but really its name should be more explicit.